modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/_ExperienceToolbarSection.scss

```diff
@@ -37,4 +39,16 @@ html#{$cadmin-selector} {
 			}
 		}
 	}
+
+	body.product-menu-open {
+		.cadmin {
+			.page-editor__toolbar-experience__dropdown-menu.toggled {
+				width: calc(100vw - (320px + 24px));
+
+				@include media-breakpoint-up(lg) {
+					width: calc(#{$experiences-dropdown-width} - 280px);
```

Comment:
I don't understand why 280px :S

Reply:
It was used to compensate the space taken by the label, inlined positioned with the select.
Trying another solution...